src/app/discussions/[id]/page.tsx

```diff
@@ -260,14 +260,15 @@ const Chats = () => {
         {showInfoCard && (
           <ContactInfo
             id={""}
-            title={"Contact info"}
+            title={`${receiver?.isGroup ? "Group info" : "Contact info"}  `}
             onClose={() => setShowInfoCard((prev) => !prev)}
             picture={
+              receiver?.image ||
               "https://i.pinimg.com/564x/fe/85/c3/fe85c35b97c3f14082ac2edfb25eba44.jpg"
             }
-            name={"Caleb matins"}
+            name={receiver?.name}
             about={"made of gold"}
-            email={"calebmatins@gmail.com"}
+            email={receiver?.email}
```

Comment:
receiver object does not have email

Reply:
and i just noticed the is no email again on the local storage